### PR TITLE
Add repo-install script

### PR DESCRIPTION
**Pull request by hectorespert**
Add repo-install script to add packages in the repo

### CODE REVIEW

File: sources/repo-insert [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+if [ ! -d "${STAGINGDIR}" ]; then
+  [ -e "${STAGINGDIR}" ] && error_mes "file" "${STAGINGDIR}"
+  echo -e "${bwhite}[II] ${bblue}Creating directory ${bwhite}'${STAGINGDIR}'${cclose}"
+  /usr/bin/mkdir -p "${STAGINGDIR}" || error_mes "unknown"
+fi
+
+
+cd "${STAGINGDIR}"
+for PKGNAME in ${PKGLIST[@]}; do
+	/usr/bin/yaourt -G ${PKGNAME}
```
Comment (arcan1s) [inline sources/repo-insert]: please use double quotes for variable

File: sources/repo-insert [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+	if [ ! -d "${PKGNAME}" ]; then
+		echo -e "${bwhite}[II]${cclose} Not found the package ${bwhite}${PKGNAME}${cclose}"
+	else
+		echo -e "${bwhite}[II]${cclose} Added package ${bwhite}${PKGNAME}${cclose}"
+		if [ ${UPDATEREPO} == "true" ]; then
```
Comment (arcan1s) [inline sources/repo-insert]: the same as previous

File: sources/repo-insert [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+			if [ "${CMD_NOCOLOR}" == "yes" ]; then
+				/usr/bin/repo-update --nocolor -c ${CONF_FILE}
```
Comment (arcan1s) [inline sources/repo-insert]: the same as previous

File: sources/repo-insert [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+			else
+				/usr/bin/repo-update -c ${CONF_FILE}
```
Comment (arcan1s) [inline sources/repo-insert]: the same as previous

File: sources/repo-insert [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+			fi
+		elif [ ${CHECKREPO} == "true" ]; then
```
Comment (arcan1s) [inline sources/repo-insert]: the same as previous

File: sources/repo-insert [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+			if [ "${CMD_NOCOLOR}" == "yes" ]; then
+				/usr/bin/repo-check --nocolor -c ${CONF_FILE}
```
Comment (arcan1s) [inline sources/repo-insert]: the same as previous

File: sources/repo-insert [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+			else
+				/usr/bin/repo-check -c ${CONF_FILE}
```
Comment (arcan1s) [inline sources/repo-insert]: the same as previous

File: sources/repo-insert [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+    "file"      ) echo "${_red}[EE] ${_bwhite}'${2}' is a file${_cclose}"              ;;
+    "unknown"   ) echo "${_red}[EE] ${_bwhite}Unknown error${_cclose}"                 ;;
+    "flag"      ) echo "${_red}[EE] ${_bwhite}Unknown flag${_cclose}"                  ;;
+  esac
+  exit 1
+}
+
+func_help() {
+  echo -e "Script written on BASH to add a package in the repository"
+  echo -e "Usage: repo-insert [ -c /etc/repo-scripts.conf ] [ -v | --version ] [ -h | --help ] PACKAGE1 PACKAGE2 ..."
```
Comment (arcan1s) [inline sources/repo-insert]: it would be great if you will update help message according to the available flags

**Comment by hectorespert** [comment]
Update help message and add double quotes for variables.
Add new message error.
Move the read configuration message to use colors

**Comment by hectorespert** [comment]
Can yo create a new release with my scripts?

**Comment by arcan1s** [comment]
of course, will do today

**Pull request merged**